pixi.toml

```diff
@@ -104,6 +104,8 @@ snowflake-connector-python = "*"
 xlrd = "*"
 xlsxwriter = "*"
 openpyxl = "*"
+datasets = "3.2.0.*"
+transformers = "4.47.1.*"
```

Comment:
I'm assuming these were for the AI examples? Are they needed at this point?

Reply:
Got added by mistake. Will remove now.

Reply:
Done.